### PR TITLE
Oso multitenant rbac

### PR DESCRIPTION
**Pull request by chinedu117**

### CODE REVIEW

File: data_model/__init__.py [hunk 1]
```diff
@@ -140,4 +144,40 @@ class Features(Base):
...
 
 
+class Users(Base):
+    __tablename__ = "users"
+    __table_args__ = {'schema': 'cdm'}
+    id = Column(  String, comment="ID / primary key", primary_key=True)
+    username = Column(String, comment="User name")
 
+
+# StationsRoleMixin = resource_role_class(
```
Comment (isedwards) [inline data_model/__init__.py]: @chinedu117 is it still possible for additional tables introduced by `oso` to be created by `resource_role_class`?

**Comment by isedwards** [comment]
@chinedu117 - I'm trying out this latest PR and having a few problems

```
1. Run `docker-compose up -d`.
2. `docker-compose run db-builder`
3. `pytest`
```

Running `pytest` in the container results in no tests being found

```
root@acbe7d2557d6:/opt/project# pytest
================================================== test session starts ==================================================
platform linux -- Python 3.10.9, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /opt/project
plugins: order-1.0.0
collected 0 items                                                                                                       

================================================= no tests ran in 0.01s =================================================
```

I can't run the build:

```
root@acbe7d2557d6:/opt/project# python build_db.py
python: can't open file '/opt/project/build_db.py': [Errno 2] No such file or directory
```

The only file in `/opt/project` is the `__init__.py`
```
root@acbe7d2557d6:/opt/project# ls
__init__.py
```

**Comment by chinedu117** [comment]
> @chinedu117 - I'm trying out this latest PR and having a few problems
> 
> ```
> 1. Run `docker-compose up -d`.
> 2. `docker-compose run db-builder`
> 3. `pytest`
> ```
> 
> Running `pytest` in the container results in no tests being found
> 
> ```
> root@acbe7d2557d6:/opt/project# pytest
> ================================================== test session starts ==================================================
> platform linux -- Python 3.10.9, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
> rootdir: /opt/project
> plugins: order-1.0.0
> collected 0 items                                                                                                       
> 
> ================================================= no tests ran in 0.01s =================================================
> ```
> 
> I can't run the build:
> 
> ```
> root@acbe7d2557d6:/opt/project# python build_db.py
> python: can't open file '/opt/project/build_db.py': [Errno 2] No such file or directory
> ```
> 
> The only file in `/opt/project` is the `__init__.py`
> 
> ```
> root@acbe7d2557d6:/opt/project# ls
> __init__.py
> ```

Probably you are on windows os, please try again i have updated the docker-compose file.

**Review by isedwards — APPROVED** [review]

**Comment by isedwards** [comment]
Thank you @chinedu117 - it is working now (I'm using Ubuntu server 22.04)

**Pull request merged**